src/components/Header/Navbar/NavbarElements.jsx

```diff
@@ -40,6 +41,7 @@ export const NavbarContainer = styled.div`
     z-index: 1;
     width: 100%;
     max-width: 1500px;
+    // display: none;
```

Comment:
if you don't use it, remove it , I am writing here, but I refer to all commented styles that you don't use.